ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.java

```diff
@@ -417,8 +417,9 @@ protected int determineActiveToolState() {
     }
 
     private int determineLanguageState(final int languageIndex) {
-        if (mManifests.get(languageIndex) != null) {
-            return STATE_LOADED;
+        Manifest manifest = mManifests.get(languageIndex);
+        if (manifest != null) {
+            return manifest.getPages().size() > 0 ? STATE_LOADED : STATE_NOT_FOUND;
```

Comment:
within the Manifest object there is a `mType` field, expose that via a `getType()` accessor and utilize that value to determine what type of tool this is